index.js

```diff
@@ -15,7 +15,11 @@ var PLUGIN_NAME = 'gulp-sass';
 var gulpSass = function gulpSass(options, sync) {
   return through.obj(function(file, enc, cb) {
     var opts,
+        ip,
         filePush,
+        sassMap,
+        sassMapFile,
+        sassFileSrc,
```

Comment:
is it the linting rule that forces to declare local variable to a if globally to the plugin?